src/Config.cs

```diff
@@ -27,6 +32,15 @@ public static void RegisterConfig()
             MelonPreferences.CreateEntry(Category, nameof(LetModsRemoveRequests), true,
                                     "Allows mods to remove requests from the queue.");
 
+            MelonPreferences.CreateEntry(Category, nameof(TwitchUsername), "",
+                                    "Twitch Username (used for writing messages back to twitch chat)");
+
+            MelonPreferences.CreateEntry(Category, nameof(TwitchAccessToken), "",
```

Comment:
Storing this as plain text is not exactly ideal, especially if it shows up in-game on the mod settings page (which I think it will?) I think you should consider moving this into a separate config file that the user has to explicitly set up. Actually, shouldn't all of this stuff be handled by the chat bot mod instead of the request mod?

Reply:
Actually it _is_ handled elsewhere..  this is probably a remnant of a previous approach.  I'll remove it.  

On a separate note, I do use MelonPreferences to store this stuff similarly, however the TwitchConnectorMod doesn't have any UI. The intent is for that information to be populated by the user by editing the MelonPreferences.cfg file directly.  Agree that oauth info shouldn't be shown in the game itself at anytime, especially for streamers.

Reply:
Removed.